lib/LaTeXML/Package/siunitx.sty.ltxml

```diff
@@ -105,14 +132,40 @@ AtBeginDocument('\lx@six@initialize');
 # Some useful macros & symbols
 DefMathI('\SIUnitSymbolCelsius',   undef, UTF(0xB0) . "C");
 DefMathI('\SIUnitSymbolOhm',       undef, "\x{2126}");
-DefMathI('\SIUnitSymbolDegree',    undef, UTF(0xB0));
-DefMathI('\SIUnitSymbolArcminute', undef, '{}^{\prime}');
-DefMathI('\SIUnitSymbolArcsecond', undef, '{}^{\prime\prime}');
+DefMathI('\SIUnitSymbolDegree',    undef, UTF(0xB0));         # , lpadding=>0 ????
+DefMathI('\SIUnitSymbolArcminute', undef, "\x{2032}");
+DefMathI('\SIUnitSymbolArcsecond', undef, "\x{2033}");
 DefMathI('\SIUnitSymbolAngstrom',  undef, UTF(0xC5));
 DefMathI('\SIUnitSymbolMicro',     undef, UTF(0xB5));
 
+# These need less left space.
+# Should these be multiplied "ID"s, or postfix?
+DefConstructorI('\SIUnitSymbolDegree', undef,
+  "<ltx:XMTok meaning='arcminute' role='ID' lpadding='0pt'>" . UTF(0xB0) . "</ltx:XMTok>");
+DefConstructorI('\SIUnitSymbolArcminute', undef,
+  "<ltx:XMTok meaning='arcminute' role='ID' lpadding='0pt'>\x{2032}</ltx:XMTok>");
+DefConstructorI('\SIUnitSymbolArcsecond', undef,
```

Comment:
Should these have both `DefMathI` and `DefConstructorI`?

Reply:
Oh, not really; I didn't finish figuring out if `DefMath` should accept `lspace` (and friends) as options.

Reply:
And, on top of that, I had misdefined them when I redefined them.  I added lpadding, rpadding to `DefMath`.

Reply:
Nice!